lib/AST/ASTContext.cpp

```diff
@@ -5724,6 +5729,35 @@ SubstitutionMap::Storage *SubstitutionMap::Storage::get(
   return result;
 }
 
+ProtocolConformanceRef ProtocolConformanceRef::forAbstract(
+    Type conformingType, ProtocolDecl *proto) {
+  ASTContext &ctx = proto->getASTContext();
+
+  // Figure out which arena this should go in.
+  RecursiveTypeProperties properties;
+  if (conformingType)
```

Comment:
Where are the remaining callers that pass a null Type?

Reply:
I don't know of any, but I want the assertions in place before I take this out.